lib/microformats/format_parser.rb

```diff
@@ -267,8 +75,8 @@ def parse(element, base: nil, element_type: nil, format_class_array: [], backcom
       h_object['children'] = @children unless @children.empty?
 
       if @format_property_type == 'e'
-        h_object['value'] = element.text.strip
-        h_object['html'] = element.inner_html
+        h_object['value'] = render_text(element)
+        h_object['html'] = element.inner_html.gsub(/\A +/, '').gsub(/ +\Z/, '')
```

Comment:
these are something that we will need to look in to closer.  The test set hasn't been terribly clear in the past on when to include what whitespaces or not.  if you look at spec/lib/microformats/parser_spec.rb    there are some 6 tests disabled because of whitespace issues.  I would rather tackle all this in a separate PR

If you look at the list of